.github/workflows/build.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Our default branch name is "master", please modify the workflow to trigger on the "master" branch instead.
@MohammadSianaki